src/pages/landing-page/index.js

```diff
@@ -136,6 +136,22 @@ const SwaprButton = styled(Button)`
     }
 `;
 
+const DxtrustButton = styled(Button)`
+    background-color: var(--violet-color);
+    color: var(--white);
+    border: 1px solid rgba(0, 0, 0, 0.075);
+    border-radius: 6px;
+    &:hover {
+        background: var(--violet-hover);
+        border: 1px solid rgba(0, 0, 0, 0.075);
+        cursor: pointer;
+    }
+    @media (max-width: 475px) {
+        width: 47%;
+        margin-right: 12px;
+    }
+`;
+
```

Comment:
Seems that code is the same as Swapr button....so in this case you should just make one styled css component